src/app/content/search/components/SearchResultsSidebar/SearchResultsBarWrapper.tsx

```diff
@@ -100,13 +101,14 @@ export class SearchResultsBarWrapper extends Component<ResultsSidebarProps> {
   </Styled.NavOl>;
 
   public render() {
-    const { results, book, searchResultsOpen, hasQuery } = this.props;
+    const { results, book, searchResultsOpen, hasQuery, dataAnalyticsRegion } = this.props;
```

Comment:
having this be here and just be a different name isn't great. the difference in name is arbitrary. you could just change this to be `data-analytics-region` and i'd probably be ok with that, but i don't love it because its explicit handling of a thing totally unrelated to this component.

alternatively you could forward all unknown props to the `SearchResultsBar`, i like that because then any html props you put in the `SearchResultsWrapper` would end up, you know, on the wrapper. in order to accomplish that you'd have to add some stuff to the destructuring here just to avoid forwarding them. the breakdown is:
- query, you'd have to add to the deconstruction so it didn't get passed down
- hasQuery, is currently being destructured, but you could remove that and let it be part of the remaining props that get forwarded
- results, no change
- onClose, add to deconstruction, its being used anyway
- searchResultsOpen, is currently being deconstructed, but you could remove that and let it be part of the remaining props that get forwarded
- book, no change
- totalHits, you'd have to add just to avoid forwarding
- selectedResult, you'd have to add just to avoid forwarding
- data-analytics-region, would get forwarded

or finally you could add a new prop, for like... `wrapperHtmlProps` or something that would get forwarded down and applied to the actual wrapper element. this is more explicit and you don't have to deliberately ignore all the props you know about. sometimes i find these kind of "html prop containers" to be kind of eh, but it is a valid pattern that i've seen before so we could follow that if you think its better